cibuildwheel/__main__.py

```diff
@@ -309,5 +309,28 @@ def detect_warnings(platform: str, build_options: BuildOptions) -> List[str]:
     return warnings
 
 
+def detect_setup_pyproject_existence(package_dir: str) -> bool:
+    """Detects whether setup.py, or pyproject.toml is at package dir
+
+    Parameters
+    ----------
+    package_dir : str
+        Package directory path
+
+    Returns
+    -------
+    bool
+        Whether setup.py, or pyproject.toml exists
```

Comment:
This seems kind of obvious, it's not part of a public API (because we don't have one), and we're not doing it anywhere else, so I'm not entirely sure it's useful/necessary.

Reply:
I'll remove it :)

Reply:
Just my two cents. The others might think it's a good idea to start doing this for everything, now ;-)